core/src/epicli/data/common/defaults/configuration/repository.yml

```
@@ -3,4 +3,11 @@ title: "Epiphany requirements repository"
 name: default
 specification:
   description: "Local repository of binaries required to install Epiphany"
-  download_done_flag_expire_minutes: 120
\ No newline at end of file
+  download_done_flag_expire_minutes: 120
+  teardown:
+    disable_http_server: true
+    remove:
+      files: true
+      helm_charts: true
+      images: true
+      packages: true 
```

```suggestion
      files: false
      helm_charts: false
      images: false
      packages: false
```
I would vote for leaving epirepo files, they can be reused for offline installation.

---

ok